app/services/katello/event_daemon.rb

```diff
@@ -0,0 +1,45 @@
+module Katello
+  class EventDaemon
+    def self.start(via_rake: false)
+      if started?
+        Rails.logger.debug("Katello Event Daemon already running, not attempting startup")
+        return
+      end
+
+      in_console = false
+      Rails.application.console do
+        Rails.logger.debug("Not starting Katello Event Daemon in a console session")
+        in_console = true
+      end
+
+      return if in_console
+
+      should_run = if via_rake
+                     true
+                   else
+                     !Foreman.in_rake? && !Rails.env.test? && SETTINGS[:katello][:with_event_monitor]
```

Comment:
i assume the setting was to allow a developer to choose to not run it?

Reply:
Yep, it's false by default and I plan to puppetize it to true for dev

Reply:
Also so that we can start rails in production mode in a dev environment, if we want, with the event monitor running which we wouldn't do in a real deployment